Functional.Primitives.Extensions/OptionExtensions.cs

```diff
@@ -142,5 +142,19 @@ public static void Apply<TValue>(this Option<TValue> option, Action<TValue> appl
 
 		public static Task Apply<TValue>(this Task<Option<TValue>> option, Action<TValue> applyWhenSome, Action applyWhenNone)
 			=> option.Do(applyWhenSome, applyWhenNone);
+
+		[AllowAllocations]
+		public static IEnumerable<T> ToEnumerable<T>(this Option<T> option)
+			=> option.TryGetValue(out var value) ? new[] { value } : Enumerable.Empty<T>();
```

Comment:
is TryGetValue better than a Match?

Reply:
`TryGetValue` is an (`internal`) extension method that avoids the allocation that would normally happen when using `Match`.  I'm also following conventions established elsewhere in this file.